corehq/apps/api/odata/utils.py

```diff
@@ -0,0 +1,13 @@
+from corehq.apps.app_manager.app_schemas.case_properties import ParentCasePropertyBuilder
+from corehq.apps.app_manager.dbaccessors import get_apps_in_domain
+from corehq.apps.reports.analytics.esaccessors import get_case_types_for_domain_es
+
+
+def get_case_type_to_properties(domain):
+    # TODO - use CaseExportDataSchema
+    apps = get_apps_in_domain(domain)
```

Comment:
this is a very expensive call, no? +1 to using `CaseExportDataSchema`

Reply:
https://github.com/dimagi/commcare-hq/pull/23207/commits/b117be087c65ee3b36288288d3745e6d65971414